<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>877</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabUI">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>861</width>
      <height>651</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tabCustomerUI">
     <attribute name="title">
      <string>Customer</string>
     </attribute>
     <widget class="QPushButton" name="btStartProgram">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>560</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Start (S)washing</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>271</width>
        <height>201</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>68</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Price:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>50</y>
         <width>68</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>7,95</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Estimated duration:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>80</y>
         <width>68</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>1 Hour</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Finished at:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>120</y>
         <width>68</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>13.45</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="btCallEmployee">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>370</y>
        <width>231</width>
        <height>151</height>
       </rect>
      </property>
      <property name="text">
       <string>Call an employee</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbWelcome">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>351</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>27</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Welcome To Swash!</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>480</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>520</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>560</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Receivelabe</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>500</y>
        <width>81</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>60</y>
        <width>511</width>
        <height>321</height>
       </rect>
      </property>
      <property name="title">
       <string>Add Garments:</string>
      </property>
      <widget class="QListWidget" name="listWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>256</width>
         <height>192</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="cmbColour">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>30</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Black</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>White</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Coloured</string>
        </property>
       </item>
      </widget>
      <widget class="QCheckBox" name="cbWash">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>150</y>
         <width>100</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Wash</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="cbDry">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>180</y>
         <width>100</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Dry</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="cbSteam">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>210</y>
         <width>100</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Steam</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="cbCentrifuge">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>240</y>
         <width>100</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Centrifuge</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cmbMaterial">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>70</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Cotton</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Silk</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Wool</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Satin</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Nylon</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Linen</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Polyester</string>
        </property>
       </item>
      </widget>
      <widget class="QDoubleSpinBox" name="nmrWeight">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>110</y>
         <width>50</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btAddGarment">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>280</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Add Garment</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tabAdministratorUI">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <attribute name="title">
      <string>Administrator</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>351</width>
        <height>251</height>
       </rect>
      </property>
      <property name="title">
       <string>Machines</string>
      </property>
      <widget class="QListView" name="listViewMachines">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>331</width>
         <height>211</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>681</width>
        <height>231</height>
       </rect>
      </property>
      <property name="title">
       <string>Garments</string>
      </property>
      <widget class="QListWidget" name="listWidgetGarments">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>331</width>
         <height>192</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="lblGarmentId">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>30</y>
         <width>301</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="lblMaterial">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>50</y>
         <width>301</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="lblWeight">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>70</y>
         <width>321</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="lblColoured">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>90</y>
         <width>321</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="lblWashingMachine">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>110</y>
         <width>321</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="lblDryer">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>130</y>
         <width>321</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="lblSteamer">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>150</y>
         <width>321</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="lblCentrifuge">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>170</y>
         <width>321</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="btStopSelectedMachine">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>400</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop Selected Machine</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btAlarmStop">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>440</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Alarm! Stop Everything</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btAddGarmentTest">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>250</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>877</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
